src/test/scala/nl.knaw.dans.easy.deposit/DepositDirSpec.scala

```diff
@@ -150,9 +165,9 @@ class DepositDirSpec extends TestSupportFixture with MockFactory {
 
     // post conditions
     mdFile.contentAsString should startWith(s"""{"identifiers":[{"scheme":"id-type:DOI","value":"12345"}]""")
-    val newDepositProperties = depositPropertiesFile.contentAsString
-    newDepositProperties should include(oldDepositProperties)
-    newDepositProperties should include("identifier.doi")
+    depositPropertiesFile.lines.toSeq
+      .diff(oldDepositProperties).mkString("\n") shouldBe
```

Comment:
Why make a String out of this? We have all kinds of list checks in ScalaTest. `should contain only (...)` is the one I would use here.